tests/test_file.py

```diff
@@ -111,56 +110,72 @@ def _test_file_put(self, replica: Replica, scheme: str, test_bucket: str, upload
                           f"when uploading a file with without UUID {invalid_uuid}"):
             self.upload_file(source_url, '', expected_code=requests.codes.forbidden)
 
+    @staticmethod
+    def _upload_file_to_mock_ingest(
+            uploader_class: typing.Type[Uploader],
+            bucket: str,
+            key: str,
+            data: bytes) -> None:
+        tempdir = tempfile.gettempdir()
+        uploader = uploader_class(tempdir, bucket)
+        with tempfile.NamedTemporaryFile(delete=True) as fh:
+            fh.write(data)
+            fh.flush()
+
+            uploader.checksum_and_upload_file(fh.name, key, "text/plain")
+
+    @testmode.standalone
+    def test_file_put_large_sync(self):
```

Comment:
Add a test for ASYNC_COPY_THRESHOLD - 1. Those are the most common areas for areas occur.

Reply:
These tests are run in parallel, so As long as it's not slower then test_indexer.

Reply:
Ah ha ha ha ha, ohhh, test_indexer...